client/src/components/Description.jsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
-import {DescriptionContainer} from '../styling.jsx';
+import {DescriptionContainer, WhyWeMadeThis, ItemDescription} from '../styling.jsx';
 
 const Description = (props) => {
   return (
     <DescriptionContainer>
-      Why we made this:
-      {props.description}
+      <WhyWeMadeThis>Why we made this: </WhyWeMadeThis><br></br>
```

Comment:
I'm curious re the reasoning behind the <br> tag here? Wouldn't this usually be handled by styling?

Reply:
I was messing around with styled-components, and I figured I could treat this as a component for clarity and customizability's sake